superset/charts/post_processing.py

```diff
@@ -171,7 +174,7 @@ def pivot_df(  # pylint: disable=too-many-locals, too-many-arguments, too-many-s
             for subgroup in subgroups:
                 slice_ = df.index.get_loc(subgroup)
                 subtotal = pivot_v2_aggfunc_map[aggfunc](
-                    df.iloc[slice_, :].apply(pd.to_numeric), axis=0
+                    df.iloc[slice_, :].apply(pd.to_numeric, errors="coerce"), axis=0
```

Comment:
i think this might be the only change needed to deal with this issue: https://github.com/apache/superset/issues/27499

Reply:
pandas is so opaque, especially when you haven't touched it for years - `.iloc[]?`, "coerce"!?!? might be worth adding a comment that explains what it's doing

Reply:
it's `loc`s and `iloc`s all the way down 🐢